ci.sh

```diff
@@ -95,7 +95,7 @@ run_old_e2e_tests() {
         # /tmp/parsec/parsec.sock. This can not be created in the Dockerfile as this is where
         # the repository is checked out.
         ln -s /tmp/parsec.sock /tmp/parsec/parsec.sock
-        RUST_BACKTRACE=1 cargo test --manifest-path /tmp/old_e2e_tests/Cargo.toml normal_tests
+        RUST_BACKTRACE=1 cargo test --manifest-path /tmp/old_e2e_tests/Cargo.toml normal_tests -- --skip asym_verify_fail
```

Comment:
Maybe add a comment explaining why we ignore this specific test?

Reply:
I'll add a comment, no one will check the CHANGELOG if they see this in the code at some point.

Reply:
I meant adding the comment in this file as part of the CHANGELOG PR, to not have to run the CI once more

Reply:
Ah, sure!